page-index/page-index.js

```diff
@@ -549,9 +549,13 @@ const pageIndex = [
                 type: LINK,
                 href: "https://github.com/timescale/timescaledb/blob/master/docs/StyleGuide.md"
             }, {
-                Title: "How to contribute",
+                Title: "Contribute to code",
                 type: LINK,
                 href: "https://github.com/timescale/timescaledb/blob/master/CONTRIBUTING.md"
+            }, {
```

Comment:
Long story, but there is a glitch in the `web-documentation` code that causes this page not to appear.  To solve the problem in the short term , you need to put the new entry ahead of the two `LINK` entries in this list of children.